pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -28,6 +28,13 @@ type StorageClusterSpec struct {
 	MonDataDirHostPath string                                 `json:"monDataDirHostPath,omitempty"`
 	// Version specifies the version of StorageCluster
 	Version string `json:"version,omitempty"`
+	Network NetworkSpec `json:"network,omitempty"`
```

Comment:
Add a comment with a short description of this spec. The comment is converted into a description to assist users.